modules/installation-three-node-cluster.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_azure/installing-azure-user-infra.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc [Eventually?]
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_ibm_z/installing-ibm-z.adoc [Eventually?]
+
+[id="installation-three-node-cluster_{context}"]
+= Running a three-node cluster
+
+You can install and run a three-node clusters in {product-title}. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for deployment, development, and testing.
+
+[IMPORTANT]
+====
+Three-node clusters are currently a Technology Preview feature.
+ifdef::openshift-enterprise,openshift-webscale[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+See the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat
+Technology Preview features support scope] for more information.
+endif::[]
+====
+
+////
+.Prerequisites
+
+* You must create at least two compute machines, which are also known as worker machines, during installation.
```

Comment:
No sure why we need extra compute/worker machines during installation. From my understanding, three-node cluster means that the three nodes/machines have both master and worker role.